seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/FileFormat.java

```diff
@@ -55,7 +55,10 @@ public ReadStrategy getReadStrategy() {
         public ReadStrategy getReadStrategy() {
             return new JsonReadStrategy();
         }
-    };
+    },
+    EXCEL("xlsx")
+    ;
```

Comment:
``` EXCEL("xlsx") {
        @Override
        public ReadStrategy getReadStrategy() {
            return new ExcelReadStrategy();
        }
```
hi, suggestion change to it, and you alson need excelReadStragey

Reply:
![image](https://user-images.githubusercontent.com/32196893/187445927-01ac825e-4526-4e40-9ede-07f5203eb7af.png)
Do I need to implement it？I see people in the community working on Excel Source

Reply:
> ![image](https://user-images.githubusercontent.com/32196893/187445927-01ac825e-4526-4e40-9ede-07f5203eb7af.png) Do I need to implement it？I see people in the community working on Excel Source

I don't think you need to implement it in this pr.